api/.gitignore

```
@@ -1 +1 @@
-
+

```

Let's add the `__pycache__` folder to the `.gitignore` so it doesn't get saved to GitHub, since it gets generated whenever you run the python scrips, but is not necessary to run the code in another system.

```suggestion
**/__pycache__/
```